index.js

```diff
@@ -1,9 +1,22 @@
-const kovanStaging = require('./kovan-staging.json');
+const fs = require('fs')
+
+function loadMetadata(folder) {
+  let pools = [];
+  fs.readdirSync(folder).forEach(function(file) {
```

Comment:
When I pull the commit of this PR into `tinlake-ui`, I get the following error: `TypeError: fs.readdirSync is not a function`. This method is a node.js method while it's now running in the browser, so that doesn't work. I think we need to run this script at compile time and then generate static .json files.

Reply:
Fixed